index.js

```diff
@@ -56,7 +57,11 @@ function openFile (file, sonic) {
 
     // start
     if (!sonic._writing && sonic._len > sonic.minLength && !sonic.destroyed) {
-      actualWrite(sonic)
+      if (sonic.contentMode === 'buffer') {
```

Comment:
Wondering if `contentMode` as an enum instead of a string would make more sense. Either way, this should be included in `types/index.d.ts` and should be documented.

Reply:
great idea!